app/Repositories/PostRepository.php

```diff
@@ -94,7 +94,7 @@ public function create(array $data, $signupId, $authenticatedUserRole = null)
             'remote_addr' => request()->ip(),
         ]);
 
-        $isAdmin = isset($data['status']) && isset(auth()->user()->role) && auth()->user()->role === 'admin';
+        $isAdmin = auth()->user()->role && auth()->user()->role === 'admin';
         $hasAdminScope = in_array('admin', token()->scopes());
```

Comment:
I don't think we need the `isset($data['status']` check here since we do this below! Also, for some reason `isset(auth()->user()->role)` was returning an error so that is why when I uploaded a photo via the uploader, the post in was marked as `pending`. I updated this and is working now!

Reply:
hmm i think we still need to check if exists. If is a machine client_id, auth role won't be available to us, just `token()->scope()`

Reply:
Yeah, I'd write this as `auth()->user() && auth()->user()->role === 'admin';`